.detekt.yml

```diff
@@ -70,6 +70,11 @@ style:
     maxDestructuringEntries: 4
   ForbiddenComment:
     active: false
+  ForbiddenMethodCall:
+    active: true
+    # Use org.ossreviewtoolkit.utils.ort.runBlocking instead in all code that can be used as a library.
+    # One exception to that are the CLI commands.
```

Comment:
IMO that comment better goes to the commit message, see my comment below.

Reply:
See above.